Power Platform extensibility options

kkazala opened the pull request:
## Category

- [ ] Content fix
- [x] New article

## Related issues

N/A

## Contents of the Pull Request

Summary of currently available extensibility options for Power Platform, including programming languages and constraints. 
The articles are accompanied by visual summary of which option can be implemented where and used by which part of PP.

LuiseFreese commented:
HI again, thank you. As this is a series, do you maybe want to not publish all of the parts at once? you can control this with the publishing date in the front matter of each md article. Also, would it be ok for you to state in the title which part this is and in the text links to the other parts?

kkazala commented:
Wow, you are fast =)

I would prefer to publish all at once, I split the article into series to make it easier to read but I think it makes more sense to publish all these posts together, 

Just added the links, not sure how to rename the tiles thought. I have
- Power Platform extensibility options
- Power Platform extensibility options in Power Apps
- Power Platform extensibility options in Power Automate
- Power Platform extensibility options in Dataverse

Do you have a suggestion? I don't want to make the tile longer than the post :D

LuiseFreese commented:
I updated the publishing date to be March20 for all of them and merged. Thank you!
